src/views/addProperty/addProperty.js

```diff
@@ -1,239 +1,180 @@
-import React, { Component } from 'react';
+import React, { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
 import { Form, Field, Formik } from 'formik';
 import * as Yup from 'yup';
 import UserContext from '../../UserContext';
 import * as axios from 'axios';
-import { Link } from 'react-router-dom';
 import Toast from '../../utils/toast';
-import { SearchPanel, SearchPanelVariant } from 'react-search-panel';
-import RoleEnum from '../../Enums/RoleEnum';
-import './addProperty.scss'
+
+import './addProperty.scss';
 
 
 const validationSchema = Yup.object().shape({
   name: Yup.string()
-    .min(5, `*Name must have at least 5 characters`)
-    .max(100, `*Names can't be longer than 100 characters`)
-    .required(`*Name is required`),
+    .min(5, "*Name must have at least 5 characters")
+    .max(100, "*Names can't be longer than 100 characters")
+    .required("*Name is required"),
   address: Yup.string()
-    .max(250, `*Address can't be longer than 250 characters`)
-    .required(`*Address is required`),
+    .max(250, "*Address can't be longer than 250 characters")
+    .required("*Address is required"),
   city: Yup.string()
-    .max(50, `*City can't be longer than 50 characters`)
-    .required(`*City is required`),
+    .max(50, "*City can't be longer than 50 characters")
+    .required("*City is required"),
   zipcode: Yup.string()
-    .max(50, `*Zipcode can't be longer than 50 characters`)
-    .required(`*Zipcode is required`),
+    .max(50, "*Zipcode can't be longer than 50 characters")
+    .required("*Zipcode is required"),
   state: Yup.string()
-    .max(50, `*State can't be longer than 50 characters`)
-    .required(`*State is required`),
+    .max(50, "*State can't be longer than 50 characters")
+    .required("*State is required"),
   units: Yup.string()
-    .max(50, `*Unit can't be longer than 50 characters`),
-  managers: Yup.array()
+    .max(50, "*Unit can't be longer than 50 characters"),
 });
 
-const formHandler = (data, context) => {
-  axios.post('/api/properties', data, { headers: { 'Authorization': `Bearer ${context.user.accessJwt}` } })
-    .then(function (response) {
-      Toast('Property Added!', 'success');
-    })
-    .catch(function (error) {
-      Toast(error, 'error');
-    })
-}
-
-export class AddProperty extends Component {
```

Comment:
So all of this needs to come back and we'll probably have to verify the rest of the code to make sure things didn't get lost due to merge conflicts.
Make sure when you are resolving merge conflicts you combine what is expected to be there in develop _then_ add in your own changes so nothing from develop gets accidentally overwritten

Reply:
> Make sure when you are resolving merge conflicts you combine what is expected to be there in develop _then_ add in your own changes so nothing from develop gets accidentally overwritten
I'm kind of frustrated, because I did this, again and again. I have no idea why this would be out of date because I literally pulled fresh files from development (overwriting my files) and reconstructed my changes to the many pages that needed the wrapper div, more than once, and I know everything was current. I'll re-do this, yet again, but you should know that I took special pains to make sure I had the latest changes incorporated when I last reviewed the files, and I did the same repeatedly when you merged other PRs (despite my asking you to look at mine first, because it touched so many files).

Reply:
Sorry, Ben, I understand merge conflicts can be frustrating. If you need any help resolving them, feel free to reach out. Also if things seem to complicated to resolve, another option is to create a fresh branch off of the most recent development and make your changes again (copy and paste just your own code from your commits), then creating a new PR